README.md

```diff
@@ -60,7 +60,65 @@ See [schema.sql](./iamspanner/schema.sql).
 
 ### 5) Annotate your gRPC methods
 
-Coming soon.
+Buf annotations for rpc method authorization are described in [annotations.proto](../proto/einride/iam/v1/annotations.proto)
+
+```proto
+/* ... */
+package your.pkg;
+
+/* ... */
+
+import "einride/iam/v1/annotations.proto";
+
+/* ... */
+
+service YourService {
+  /* ... */
+
+  rpc YourMethod(YourMethodRequest)
+    returns YourMethodResponse {
+      /* ... */
+
+      option (einride.iam.v1.method_authorization) = {
+        permission: "namespace.entity.method"
+        before: {
+          expression: "test(caller, request.entity)" // iamcel expression
+          description: "The caller must have method permission against the entity"
+        }
+      };
+
+      /* ... */
```

Comment:
IMO drop these comments

Reply:
Added them for consistency, since the above examples included such comments. I'll drop them.

Reply:
Thanks!